corehq/form_processor/backends/sql.py

```diff
@@ -0,0 +1,2 @@
+class XFormSql(object):
```

Comment:
essentially this is our sql adapter where i'd implement the function required for the interface to work